gradle-jdks/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
 
     implementation 'com.palantir.baseline:gradle-baseline-java'
     implementation 'com.palantir.gradle.utils:lazily-configured-mapping'
+    implementation 'com.google.guava:guava'
```

Comment:
Could we avoid using guava, if possible? It really does seem to cause an outsized number of issues for external people when in buildscripts, at least in other plugins.

Reply:
Actually, we depend on `gradle-baseline-java` which pulls in guava already, so I guess the damage is already done.